apps/admin_api/lib/admin_api/v1/controllers/admin_user_controller.ex

```diff
@@ -51,6 +52,72 @@ defmodule AdminAPI.V1.AdminUserController do
 
   def get(conn, _), do: handle_error(conn, :missing_id)
 
+  @doc """
+  Creates a new admin user.
+
+  The requesting user must have the permissions to create admin users.
+  """
+  @spec create(Plug.Conn.t(), map()) :: Plug.Conn.t()
+  def create(conn, %{"email" => email, "redirect_url" => redirect_url} = attrs)
+      when not is_nil(email) and not is_nil(redirect_url) do
+    with {:ok, _} <- authorize(:create, conn.assigns, %User{global_role: attrs["global_role"]}),
+         attrs <- Originator.set_in_attrs(attrs, conn.assigns),
+         attrs <- Map.put(attrs, "is_admin", true),
+         {:ok, redirect_url} <- UserGate.validate_redirect_url(redirect_url),
+         {:ok, _invite} <- UserGate.invite_global_user(attrs, redirect_url) do
+      render(conn, :empty, %{success: true})
```

Comment:
Shouldn't it return the user?

Reply:
It works in the same way as before, just returning success because the user is pending. I didn't want to change the user gate to get the user back.